code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -158,6 +158,9 @@
 			var/obj/item/gun/energy/gun = module
 			if(!gun.chambered)
 				gun.recharge_newshot() //try to reload a new shot.
+		else if(istype(module, /obj/item/soap))
+			var/obj/item/soap/soap = module
+			soap.uses = 300
```

Comment:
Where does the number 300 come from?

Reply:
its the default number of uses for the cyborg's soap